src/middlewared/middlewared/plugins/cron.py

```diff
@@ -41,14 +48,38 @@ def cron_extend(self, data):
                     'Specified user does not exist'
                 )
 
+        command = data.get('command')
+        if not command:
+            verrors.add(
+                f'{schema}.command',
+                'Please specify a command for cronjob task.'
+            )
+        else:
+            allowed_length = 1000 - len(' '.join(
+                filter(
+                    bool, (
+                        f'PATH="{await self.cronjob_path()}"',
+                        '> /dev/null' if data['stdout'] else '',
+                        '2> /dev/null' if data['stderr'] else ''
+                    )
+                )
+            ))
```

Comment:
Seems we are repeating ourselves here, arent we?
Why not make a private method to get the full command, including PATH and use it in cron generation?

Then `cronjob_path` can also go away as i dont see that being useful for public API.